mobile/src/screens/CategoryScreen.jsx

```diff
@@ -0,0 +1,67 @@
+import { Image, ScrollView, Text, View } from "react-native"
+import Page from "../components/Page"
+import React, { useEffect, useState } from "react"
+import { getCategoryById } from "../utils/categoriesServices"
+import ErrorScreen from "./ErrorScreen"
+import errors from "../utils/constants/errors"
+import mobileConfig from "../../mobileConfig"
+import ProductCard from "../components/ProductCard"
+
+const CategoryScreen = props => {
+  const {
+    navigation,
+    route: {
+      params: { id }
+    }
+  } = props
+
+  const [category, setCategory] = useState({
+    products: []
+  })
+
+  const [hasError, setError] = useState(false)
+
+  useEffect(() => {
+    ;(async () => {
+      try {
+        const result = await getCategoryById(id)
+        setCategory(result.data)
+      } catch (e) {
+        setError(true)
+      }
+    })()
+  }, [id])
+
+  if (hasError) {
+    return <ErrorScreen navigation={navigation} text={errors.category.unableToLoad} />
+  }
+
+  return (
+    <Page navigation={navigation}>
+      <ScrollView className="relative">
+        <View>
+          <Image
+            style={mobileConfig.images.fullScreenImageXL}
+            source={{
+              uri: category.thumbnailURL
```

Comment:
You've written L `uri` insteda of `URL`

Reply:
Done thx. I thought it won't work but it did